Runtime/Scripts/Drone/DroneLoadController.cs

```diff
@@ -23,20 +28,21 @@ public class DroneLoadController: MonoBehaviour
     [Header("Tracking")]
     [Tooltip("An object to follow")]
     public Transform TrackingTargetTF;
-    
-    
+
+
     [Header("Load")]
     [Tooltip("The rope object that this drone is expected to get connected, maybe. Will be used to check for attachment state and such.")]
     public Transform Rope; // TODO remove this requirement.
     [Tooltip("If true, instead of tracking the target object, drone will first track the buoy and when attached to it, make the LoadLinkTF track the target.")]
     public bool AttackTheBuoy = false;
     [Tooltip("The position of where the load is attached to the rope. rope_link on SAM")]
     public Transform LoadLinkTF; // The position of the AUV is taken at the base of the rope
+    [Tooltip("Defines whether controller is in a load control state (when true), or in position tracking (when false)")]
+    public bool LoadControl;
 
     [Header("Props")]
-    public Transform PropFR;
-    public Transform PropFL, PropBR, PropBL;
-    
+    public Transform PropFR, PropFL, PropBR, PropBL;
```

Comment:
look at the editor :)

Reply:
Please expand, looks the same to me in editor inspector

Reply:
The header is gone in your image :D

Reply:
Roger all my headers seem to be gone, so something else is also going on